jwt-bcp/draft-ietf-oauth-jwt-bcp.md

```diff
@@ -180,9 +184,11 @@ not be provided by libraries, or until they are), and
 
 ## Conventions used in this document
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
-interpreted as described in {{RFC2119}}.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
```

Comment:
Please replace the whole section (minus the title) by `{::boilerplate bcp14}`

You might need to update to a recent version of kramdown-rfc2629.

Reply:
Will do